spec/features/projects/project_settings_page.rb

```diff
@@ -35,16 +35,10 @@ def initialize(project)
   end
 
   def visit_settings
-    visit settings_path
+    visit settings_project_path(@project, tab: 'custom_fields')
```

Comment:
Now the method does something different than it's name suggests. It goes specifically to the custom_fields tab instead of going to the general settings.